Modules/Core/Common/src/itkMultiThreaderBase.cxx

```diff
@@ -334,34 +338,42 @@ MultiThreaderBase::GetGlobalDefaultNumberOfThreads()
 ThreadIdType
 MultiThreaderBase::GetGlobalDefaultNumberOfThreadsByPlatform()
 {
+#if defined(ITK_HAS_SCHED_GETAFFINITY)
+  cpu_set_t mask;
+  if (sched_getaffinity(0, sizeof(cpu_set_t), &mask) !-1)
+  {
+    return static_cast<ThreadIdTYpe>(CPU_COUNT(&mask));
```

Comment:
`ThreadIdTYpe` seems to have wrong capitalization.

Reply:
Must be a `TYpo` 😸

Reply:
Ugh, the linux system I was using is down right now. I though it would be easy enough to finish this PR with out it. 😓